roles/node_prep/vars/main.yml

```diff
@@ -18,8 +15,7 @@ el8_packages:
   - nm-connection-editor
   - libsemanage-python3
   - policycoreutils-python3
-  - "{{ (check_url.status == -1) | ternary('python3-crypto','https://trunk.rdoproject.org/rhel8-master/deps/latest/Packages/python3-crypto-2.6.1-18.el8ost.x86_64.rpm') }}"
-  - "{{ (check_url.status == -1) | ternary('python3-pyghmi','https://trunk.rdoproject.org/rhel8-master/deps/latest/Packages/python3-pyghmi-1.0.22-2.el8ost.noarch.rpm') }}"
+  - python3-pyghmi
```

Comment:
Can I suggest to refactor this list to be alphabetical?
`- ipmitool
- jq
- libsemanage-python3
- libvirt
- NetworkManager-libnm
- nm-connection-editor
- policycoreutils-python3
- python3-devel
- python3-libvirt
- python3-lxml
- python3-pyghmi
- python3-yaml
- qemu-kvm
- tar`